.github/workflows/test.yml

```diff
@@ -146,6 +146,73 @@ jobs:
           path: log/
         if: always()
 
+###########################################################
```

Comment:
Please, add this to the description information of this action (at the head of this file)